bundles/org.eclipse.swt/Eclipse SWT/common/org/eclipse/swt/widgets/Event.java

```diff
@@ -376,4 +376,4 @@ void setLocationInPixels (int x, int y) {
 public String toString () {
 	return "Event {type=" + type + " " + widget + " time=" + time + " data=" + data + " x=" + x + " y=" + y + " width=" + width + " height=" + height + " detail=" + detail + "}";  //$NON-NLS-1$//$NON-NLS-2$ //$NON-NLS-3$ //$NON-NLS-4$ //$NON-NLS-5$ //$NON-NLS-6$ //$NON-NLS-7$
 }
-}
+}
```

Comment:
Please revert this unrelated/unnecessary change.

Reply:
Done